packages/commons/src/events/AdvancedSearchConfig.ts

```diff
@@ -90,13 +90,7 @@ export const FieldConfigSchema = BaseField.extend({
       `Sometimes there might be need to search a value against multiple field of same FormField type. For example
       search Country, Province, District against child.address.private and child.address.other. In such case, we
       add a one field as fieldId, and accomodate others in alternateFieldIds`
-    ),
-  excludeInSearchQuery: z.boolean().default(false).optional()
```

Comment:
This PR removes the `excludeInSearchQuery` completely as it wasn't used by any other fields than the "type of location" selection. 
Generally speaking, there should never be a reason to not add a dependency value in queries

If you have country + state inputs, at the time of creating a record, to be able to select "New York", you need to have selected "United States" as the country. When searching, both terms can be used as they always go together.